docs/wiki/framework/refuel-framework.md

```diff
@@ -103,7 +103,7 @@ The jerry can will now have a volume of 200 liters.
 0  | `fuelTruck` | My fuel truck object
 1  | `428` | New fuel supply
 
-### 2.4 Make an object into a refuel source
+### 2.4 Make an object into a refuel source (Added in ACE3 3.11.0)
```

Comment:
I'd say add it below title, because title gets added to URL when selected. Making it longer and if that note gets changed will break the old URL.

Reply:
copy-paste from https://github.com/acemod/ACE3/pull/5432/files#diff-0244aff97119e1fcfc9573cf28815a4aR146
IMO it doesn't make sense to add such note. We really should merge it after 3.11 release only. This note will be superfluous after release.

Reply:
Yeah that one is not something I like either. Someone might want to run an older version of ACE3 (eg. Linux/Mac branch), so having a note is good. We also have notes per page on top of every page.